src/test/java/auto_ckz/domain/car/CarTest.java

```diff
@@ -0,0 +1,144 @@
+        Car carEntries = repository.findByRegistrationNumber("GA5424");
+
+
+        assertThat(carEntries, allOf(
+                hasProperty("id", is(1L)),
+                hasProperty("make", is("Audi")),
+                hasProperty("model", is("A8")),
+                hasProperty("year", is(2006)),
+                hasProperty("registrationNumber", is("GA5424")),
+                hasProperty("vin", is("HF3JF4JVJ36JVJEKD")),
+                hasProperty("dateOfFirstRegistration", notNullValue()),
+                hasProperty("oc", is("6346gf234")),
+                hasProperty("vehicleCheckup", is(true)),
+                hasProperty("vehicleMileage", is(200000)),
+                hasProperty("engineCapacity", is("2200")),
+                hasProperty("enginePower", is("198")),
+                hasProperty("fuelType", is("diesel"))
+        ));
+    }
+
+    @Test
+    public void findByVin_ShouldReturnOneCarEntry() throws ParseException {
+        Car carEntries = repository.findByVin("HF3JF4JVJ36JVJEKD");
+
+
+        assertThat(carEntries, allOf(
+                hasProperty("id", is(1L)),
+                hasProperty("make", is("Audi")),
+                hasProperty("model", is("A8")),
+                hasProperty("year", is(2006)),
+                hasProperty("registrationNumber", is("GA5424")),
+                hasProperty("vin", is("HF3JF4JVJ36JVJEKD")),
+                hasProperty("dateOfFirstRegistration", notNullValue()),
+                hasProperty("oc", is("6346gf234")),
+                hasProperty("vehicleCheckup", is(true)),
+                hasProperty("vehicleMileage", is(200000)),
+                hasProperty("engineCapacity", is("2200")),
+                hasProperty("enginePower", is("198")),
+                hasProperty("fuelType", is("diesel"))
+        ));
+    }
+
+    @Test
+    public void findByOc_ShouldReturnOneCarEntry() throws ParseException {
+        Car carEntries = repository.findByOc("6346gf234");
+
+
+        assertThat(carEntries, allOf(
+                hasProperty("id", is(1L)),
+                hasProperty("make", is("Audi")),
+                hasProperty("model", is("A8")),
+                hasProperty("year", is(2006)),
+                hasProperty("registrationNumber", is("GA5424")),
+                hasProperty("vin", is("HF3JF4JVJ36JVJEKD")),
+                hasProperty("dateOfFirstRegistration", notNullValue()),
+                hasProperty("oc", is("6346gf234")),
+                hasProperty("vehicleCheckup", is(true)),
+                hasProperty("vehicleMileage", is(200000)),
+                hasProperty("engineCapacity", is("2200")),
+                hasProperty("enginePower", is("198")),
+                hasProperty("fuelType", is("diesel"))
+        ));
+    }
+
+    @Test
+    public void twoFindByMake_ShouldReturnAListOfTwoEntries() {
+        List<Car> carEntries = repository.findByMake("Audi");
+        assertThat(carEntries.size(), is(2));
```

Comment:
IMHO xml jest tworzony pod testy, po co taka informacja, tak jak w nazwie metody jest opisane: twoFindByMake_ShouldReturnAListOfTwoEntries(), powinny być wpisane w xml-u dwa tego typu dane, testy te są po to, aby sprawdzić poprawność metod, typów danych a potem ew. zgłosić to twórcom, po co taka zabawa, przecież wiadomo ocb